html-web-page/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+
+    #tagline {
+      margin-top: 10px;
+      margin-bottom: 10px;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- your HTML code here 👇 -->
+  <h1>Jelly of the Month Club</h1>
+  <h2>Make all your friends jelly of your stock of jelly!</h2>
+  <p>Here at JOTMC Inc., we are constantly striving to supply our customers
+     with the highest-quality and most unique jellies on a monthly basis. For the
+     jelly lover in your life (or yourself!), this is the gift that keeps on giving
+     all year round. We offer three tiers of monthly jelly joy, and each one offers
+     their own perks. Here is a breakdown of each, as well as your monthly price:
+  </p>
+  <div>
+    <table>
+      <thead>
+        <tr>
+          <td></td>
+          <td>
+            Basic Plan
+          </td>
+          <td>
+            Deluxe Plan
+          </td>
+          <td>
+            Jelly Giant Plan
+          </td>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per Month</td>
+          <td>&check;</td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Monthly jellies come with commemorative mason jars</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Free bonus jar of preserves bi-monthly</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Free yearly tickets to Jelly Con</td>
+          <td></td>
+          <td></td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Monthly Pricing</td>
+          <td>
+            $30
+          </td>
+          <td>
+            $50
+          </td>
+          <td>
+            $60
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <div id="tagline">
+    <label for="plan">Like what you see? Become a part of the JOTMC family today!</label>
```

Comment:
This isn't labelling any specific input in the form, so it wouldn't be a `label` element. You should make this a `p` element, which will remove the need for the additional CSS.